tensorflow/training/docker/1.15/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,224 @@
+FROM nvidia/cuda:11.0-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="7"
```

Comment:
This can't be "7", because the cu100 dockerfile is already at "7". Please change this to 8.

Reply:
Updated